catalog/app/containers/Bucket/PackageDialog/MetaInput.tsx

```diff
@@ -284,7 +284,7 @@ export const MetaInput = React.forwardRef<HTMLDivElement, MetaInputProps>(
               onChange(contents)
             } else {
               try {
-                JSON.parse(contents as string)
+                onChange(JSON.parse(contents as string))
```

Comment:
To actually fix this, I would rewrite this as two steps: first returns the result consistently, and then call `onChange` once